packages/@webex/plugin-meetings/src/roap/request.ts

```diff
@@ -143,6 +160,14 @@ export default class RoapRequest extends StatelessWebexPlugin {
       LoggerProxy.logger.info('Roap:request#sendRoap --> Race Condition /call mediaID not present');
     }
 
+    const reachabilityData = this.attachReachabilityData({
```

Comment:
the name `reachabilityData` is not great, becuase this is in fact an SDP with audio/video mute information and reachability data, not sure what a better name would be though... maybe localSdpWithReachabilityData?

Reply:
refactored